opencog/benchmark/atomspace_bm.cc

```diff
@@ -25,6 +27,8 @@ int main(int argc, char** argv)
      "          \t(default: 100000)\n"
      "-r <int>  \tLooping count; how many times a python/scheme operation is looped\n"
      "          \t(default: 1)\n"
+     "-R <int>  \tUse specific randomseed; useful for benchmark comparisons\n"
+     "          \t(default: time(NULL))\n"
```

Comment:
I didn't introduce the default to `time()` as randseed, I preserved what was there. But I can certainly see how most of the time it would be more useful the other way. We could default it to some fixed large number and then have a flag to set it to time() for sensitivity testing in the future. But I'd probably add that with support for multiple test runs and some distribution statistics in columnar display to get some sense for the dependence on random sequences.

At some point, it probably makes more sense to build some type histograms from actual data sets which we can load so we can run some tests that reflect NLP usage and other tests that reflect other OpenCog usages more directly.